### PR TITLE
Amina/ dbvi milestone 2 changes to diel

### PR DESCRIPTION
**Pull request by amina-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/appstore/src/modules/trading-hub/index.tsx [hunk 1]
```diff
@@ -284,9 +283,8 @@ const TradingHub: React.FC = () => {
                 <JurisdictionModal context={store} openPasswordModal={openRealPasswordModal} />
                 <CFDPasswordModal context={store} platform={platform} />
                 <CFDDbviOnBoarding context={store} />
-                <CFDPersonalDetailsModal context={store} />
                 <CFDResetPasswordModal context={store} platform={platform} />
-                <CFDServerErrorDialog context={store} />
+                <CFDServerErrorDialog />
```
Comment (mahdiyeh-deriv) [inline packages/appstore/src/modules/trading-hub/index.tsx:287]: for now we reverted back the context due to this commit: https://github.com/binary-com/deriv-app/pull/7311/commits/6a5293c57c2e2c030a6db2fe67d688afb57b1bf7

File: packages/cfd/src/sass/cfd-dashboard.scss [hunk 2]
```diff
@@ -2365,5 +2369,8 @@
         align-items: center;
     }
 }
+.da-icon-with-message__button {
+    margin: '2rem';
```
Comment (mahdiyeh-deriv) [inline packages/cfd/src/sass/cfd-dashboard.scss:2373]:
```suggestion
    margin: 2rem;
```

File: packages/appstore/src/modules/trading-hub/index.tsx (hunk 1 above)
Comment (amina-deriv) [reply packages/appstore/src/modules/trading-hub/index.tsx:287]: @mahdiyeh-deriv , we still pass context as a prop to `CFDServerErrorDialog`. while removing context i am getting below error. 
_Property 'context' is missing in type '{}' but required in type 'CFDServerErrorDialogProps'.

could you pls advice?

Comment (mahdiyeh-deriv) [reply packages/appstore/src/modules/trading-hub/index.tsx:287]: we need to pass the context, we will use the farzin's approach later on, for now we should pass context same as before

**Review by mahdiyeh-deriv — APPROVED** [review]

**Pull request merged**